report-viewer/vue.config.js

```diff
@@ -11,5 +11,6 @@ module.exports = defineConfig({
         chunks: "all",
       },
     },
+    devtool: 'source-map'
```

Comment:
What is this for?

Reply:
Used for debugging the report viewer. I also wondered why it was not already there.